test/Microsoft.Windows.ComputeVirtualization.Tests/ComputeTests.cs

```diff
@@ -13,7 +13,7 @@ public class ServercoreContainerTests : IDisposable
 
         public ServercoreContainerTests()
         {
-            sandbox = new Sandbox(Sandbox.ContainerType.ServerCore, id);
+            sandbox = new Sandbox(Sandbox.ContainerType.NanoServer, id);
```

Comment:
Odd that the class is "ServerCoreContainerTests" but you changed this to "NanoServer"